### PR TITLE
Add support for JUMITECH J2182548

### PR DESCRIPTION
**Pull request by kriip**
Add support for yet another (Sunricher?) OEM zigbee light dimmer:
JUMITECH J2182548
https://www.proshop.dk/Smart-Home/Jumitech-Zigbee-Dimmer/2938989

### CODE REVIEW

File: devices/jumitech.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+
+module.exports = [
+    {
+        zigbeeModel: ['J2182548'],
+        model: 'J2182548',
```
Comment (Koenkk) [inline devices/jumitech.js:7]: According to https://www.proshop.dk/Smart-Home/Jumitech-Zigbee-Dimmer/2938989 the model is `5700002182548` could you update the model here and change the file names in https://github.com/Koenkk/zigbee2mqtt.io/pull/1251

Comment (kriip) [reply devices/jumitech.js:7]: Hi @Koenkk,
I do see 5700002182548 on the barcode of the product, but everything else uses J2182548 .. are you sure we need 5700002182548?

Log from zigbee2mqtt before I created external converter:
```
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: Device '0xcc86ecfffeeecca1' joined
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xcc86ecfffeeecca1","ieee_address":"0xcc86ecfffeeecca1"},"type":"device_joined"}'
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0xcc86ecfffeeecca1"},"type":"device_connected"}'
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: MQTT publish: topic 'zigbee2mqtt/0xcc86ecfffeeecca1/availability', payload 'online'
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: Starting interview of '0xcc86ecfffeeecca1'
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xcc86ecfffeeecca1","ieee_address":"0xcc86ecfffeeecca1","status":"started"},"type":"device_interview"}'
Feb 19 11:19:58 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:19:58: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0xcc86ecfffeeecca1"},"type":"pairing"}'
Feb 19 11:20:00 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:20:00: Successfully interviewed '0xcc86ecfffeeecca1', device has successfully been paired
Feb 19 11:20:00 zigbee2mqtt npm[157]: #033[33mZigbee2MQTT:warn #033[39m 2022-02-19 11:20:00: Device '0xcc86ecfffeeecca1' with Zigbee model 'J2182548' and manufacturer name 'JUMITECH APS' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
Feb 19 11:20:00 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:20:00: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":null,"friendly_name":"0xcc86ecfffeeecca1","ieee_address":"0xcc86ecfffeeecca1","status":"successful","supported":false},"type":"device_interview"}'
Feb 19 11:20:00 zigbee2mqtt npm[157]: #033[32mZigbee2MQTT:info #033[39m 2022-02-19 11:20:00: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"friendly_name":"0xcc86ecfffeeecca1","supported":false},"type":"pairing"}'
```

Comment (kriip) [reply devices/jumitech.js:7]: External converter for completeness:

```
root@zigbee2mqtt:/opt/zigbee2mqtt/data# cat jumitech.js 
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
	zigbeeModel: ['J2182548'],
	model: 'J2182548',
	vendor: 'JUMITECH APS',
	description: 'ZigBee AC phase-cut dimmer single-line',
	extend: extend.light_onoff_brightness({noConfigure: true}),
	configure: async (device, coordinatorEndpoint, logger) => {
		await extend.light_onoff_brightness().configure(device, coordinatorEndpoint, logger);
		const endpoint = device.getEndpoint(1);
		await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
	await reporting.onOff(endpoint);
	},
};
```

Comment (Koenkk) [reply devices/jumitech.js:7]: > I do see 5700002182548 on the barcode of the product, but everything else uses J2182548 .. are you sure we need 5700002182548?

J2182548  also fine by me. Thanks!

**Pull request merged**